config/dev/objects/Campaign.object

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <webLinks>
+        <fullName>DO_NOT_PACKAGE_Household_Mailing_List</fullName>
+        <availability>online</availability>
+        <description>Deduplicated mailing list of Households</description>
+        <displayType>button</displayType>
+        <encodingKey>UTF-8</encodingKey>
+        <height>600</height>
+        <linkType>url</linkType>
+        <masterLabel>[NPSP] Household Mailing List</masterLabel>
+        <openType>sidebar</openType>
+        <protected>false</protected>
+        <url>/apex/HH_CampaignDedupeBTN?id={!Campaign.Id}</url>
+    </webLinks>
+</CustomObject>
```

Comment:
Would it be worth including a comment indicating the package where the original button is defined so it's easier to track them down to check for updates later?